logging/log_collection_forwarding/configuring-log-forwarding.adoc

```diff
@@ -13,8 +13,7 @@ include::modules/cluster-logging-collector-log-forwarding-about.adoc[leveloffset
 
 include::modules/logging-create-clf.adoc[leveloffset=+1]
 
-// uncomment for 5.9 release
-// include::modules/logging-delivery-tuning.adoc[leveloffset=+1]
+include::modules/logging-delivery-tuning.adoc[leveloffset=+1]
```

Comment:
You might need to rebase this PR since I already uncommented this in a PR that was merged earlier, just to make sure you don't get conflicts.